aiven/client/argx.py

```diff
@@ -33,6 +33,22 @@ def default(self, o):  # pylint:disable=E0202
         return jsonlib.JSONEncoder.default(self, o)
 
 
+class CustomFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescriptionHelpFormatter):
+    """Help formatter to display the default value only for integers and non-empty strings"""
+    def _get_help_string(self, action):
+        help_text = action.help
+        if '%(default)' not in action.help:
+            if action.default is not argparse.SUPPRESS:
+                defaulting_nargs = [argparse.OPTIONAL, argparse.ZERO_OR_MORE]
+                if action.option_strings or action.nargs in defaulting_nargs:
+                    if (
```

Comment:
All four nested if statements could be combined into one. If that is too unwieldy, consider using `if all([<first if condition>, <second if condition>, <third>, <fourth>]):`

Reply:
Simplified, but still 3 separate if clauses left